fastestimator/backend/_focal_loss.py

```diff
@@ -53,7 +99,8 @@ def focal_loss(y_true: Tensor, y_pred: Tensor, gamma: float = 2.0, alpha: float
                 positive vs negative examples or (-1/None) to ignore. Default = 0.25
         gamma: Exponent of the modulating factor (1 - p_t) to
                balance easy vs hard examples.
-        from_logits: Whether y_pred is logits (without softmax).
+        normalize: Whether to normalize along the batch size.
```

Comment:
Be more specific about what this normalization is doing.   Otherwise people don't know what's the difference between normalization and reduction.  

From the code it seems like the normalize is on a per-sample basis